### PR TITLE
Smaller fixes

### LINKED ISSUE #1265: revision history: start from revision not working for pages
The edit button in the revision history of pages doesn't use the correct link and therefore fails.

### LINKED ISSUE #1273: Click on course in taxonomy opens also trashed pages
Steps to reproduce:

* Go to "https://de.serlo.org/lerntipps/135480/umgang-mit-stress"
* Open the course "Einführung in den Umgang mit Stress"
* On gets a site with the message "Dieser Inhalt wurde gelöscht"

image (1)

For the course the first page is marked as trashed. I guess this page instead of the first non-trashed page is opened. See also https://serlo.slack.com/archives/C0BMSC431/p1631307758003100

### PR DESCRIPTION
**Pull request by elbotho**
- closes #1265
- closes #1273 
- decrease bundle size
- try different reload approach for `/entity/unrevised`

### CODE REVIEW

**Comment by elbotho** [comment]
@kulla I tried to simplify the down arrow by styling a unicode character (↓).
old:
image
new:
image

Do you think this works. Does it work on your setup?
e.g. https://frontend-git-smaller-fixes-serlo.vercel.app/202374

**Comment by elbotho** [comment]
@kulla and @Entkenntnis if there are no objections I'll merge this to further test the `/entity/unrevised` reload.

File: src/helper/mutations.ts [hunk 1]
```diff
@@ -116,8 +116,8 @@ export function useRevisionMutation() {
       }, 200)
       setTimeout(() => {
         NProgress.done()
-        void router.push('/entity/unrevised')
-      }, 3000)
+        void router.push('/entity/unrevised#profile-refresh')
+      }, 4000)
```
Comment (kulla) [inline src/helper/mutations.ts]: Why is there a timeout of 4000 seconds? The API cache should be cleared when the mutation is finished and will only be recomputed with the API request.

Comment (elbotho) [reply src/helper/mutations.ts]: good point. will remove if if works at all :)

**Comment by Entkenntnis** [comment]
@elbotho Feel free to merge this PR, some changes are maybe not necessary if we use https://github.com/serlo/frontend/pull/1279

**Comment by kulla** [comment]
> Do you think this works. Does it work on your setup?
> e.g. https://frontend-git-smaller-fixes-serlo.vercel.app/202374

Ping @kathongi For it is fine.

**Comment by kulla** [comment]
> @kulla and @Entkenntnis if there are no objections I'll merge this to further test the `/entity/unrevised` reload.

I am fine with merging.

**Comment by kathongi** [comment]
> @kulla I tried to simplify the down arrow by styling a unicode character (↓).
> old:
> image
> new:
> image
> 
> Do you think this works. Does it work on your setup?
> e.g. https://frontend-git-smaller-fixes-serlo.vercel.app/202374

Looks good to me :+1:

**Pull request merged**